src/Planners/src/ConvexHullHelper.cpp

```diff
@@ -5,11 +5,10 @@
  * distributed under the terms of the BSD-3-Clause license.
  */
 
-#include <libqhullcpp/PointCoordinates.h>
-#include <libqhullcpp/Qhull.h>
-#include <libqhullcpp/QhullFacetList.h>
+#include <libqhull_r/libqhull_r.h>
```

Comment:
```suggestion
extern "C"
{
#include <libqhull_r/libqhull_r.h>
}
```

See https://github.com/ami-iit/bipedal-locomotion-framework/pull/713#issuecomment-1658386129 for the rationale.

Reply:
Thank you! Let's see if the CI is happy now